packages/ps/test/commands/wait.test.ts

```diff
@@ -0,0 +1,122 @@
+import {expect, test} from '@oclif/test'
+
+const API_HOST = 'https://api.heroku.com'
+const APP_NAME = 'wubalubadubdub'
+
+const CURRENT = {
+  id: '00000000-0000-0000-0000-000000000002',
+  version: 23
+}
+
+const PREVIOUS = {
+  id: '00000000-0000-0000-0000-000000000001',
+  version: 22
+}
+
+const withRelease = test
+  .nock(API_HOST, api => api
+    .get(`/apps/${APP_NAME}/releases`)
+    .reply(200, [CURRENT])
+  )
+
+describe('heroku ps:wait', () => {
```

Comment:
These tests cover the functionality, but I have two questions:

 * how do I mock `cli.wait`?
 * how do I mock `cli.action`?